sentence_transformers/SentenceTransformer.py

```diff
@@ -2,14 +2,12 @@
 import logging
 import os
 import shutil
-import stat
 from collections import OrderedDict
 from typing import List, Dict, Tuple, Iterable, Type, Union, Callable, Optional
-import requests
 import numpy as np
 from numpy import ndarray
 import transformers
-from huggingface_hub import HfApi, HfFolder, Repository, hf_hub_url, cached_download
+from huggingface_hub import HfApi, HfFolder, upload_folder
```

Comment:
Unfortunately we can't use `create_commit`. It was just added in 0.8.1, but ST uses any version from https://github.com/UKPLab/sentence-transformers/blob/master/setup.py#L31, specially to support Python 3.6 which we don't support in hhub in latest versions (https://github.com/UKPLab/sentence-transformers/commit/39ab855919652846cb5e15c260dc9f07ecf52951)